Dockerfile

```diff
@@ -5,11 +5,13 @@ ARG DATABASE_TYPE
 ENV DATABASE_URL "postgresql://umami:umami@db:5432/umami" \
     DATABASE_TYPE=$DATABASE_TYPE
 
-COPY . /app
 WORKDIR /app
+EXPOSE 3000
 
-RUN npm install && npm run build
+COPY package.json yarn.lock /app/
```

Comment:
Should we be pulling in `package.json` here as well?

From the [yarn documentation](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install) it looks like it will fall back to `package.json` if it can't satisfy dependencies with `yarn.lock`, but we should probably be adding all new dependencies to `yarn.lock` via `yarn add`.

Reply:
yeah

the yarn.lock should always be kept satisfied when doing a commit, or it makes no sense to commit lock file.

I will double check whether the lock file is updated.

Reply:
@hugomd yes, the lock file is updated.